src/zoid/buttons/component.jsx

```diff
@@ -948,6 +949,13 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
         value: () => sessionState,
       },
 
+      getShopperInsightsUsed: {
+        type: "function",
+        value: () => {
+          return () => wasShopperInsightsUsed();
+        },
+      },
```

Comment:
this is totally a nit so ignore if you like it the way you wrote it.

```suggestion
        value: () => wasShopperInsightsUsed,
      },
```

Reply:
Good nit 🚀